runbotics/runbotics-cli/src/commands/version/version.ts

```diff
@@ -0,0 +1,98 @@
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+
+import getLocalRbConfig from './local-config';
+import verifyRemoteRbConfig from './remote-config';
+import { PRERELEASE_ID, PRERELEASE } from './utils';
+import { VersionCommandOptions, VersionOptions } from './types';
+import { Emoji } from '../utils/emoji';
+import alterFiles from './alter-files';
+import versionControl from './version-control';
+
+const { prompt } = inquirer;
+
+const getBumpType = (versionOptions: VersionOptions) => {
+    const providedOptions = Object.entries(versionOptions).filter(([_, value]) => value === true);
+
+    if (providedOptions.length > 1) {
+        console.log(chalk.red(`${Emoji.error} Error: Options [${providedOptions.map(([key, _]) => `--${key}`).join(', ')}] cannot be used together`));
+        process.exit(1);
+    }
+
+    if (providedOptions.length === 0) return PRERELEASE;
+
+    const [selectedOption, _] = providedOptions[0];
+    return selectedOption;
+};
+
+const getCurrentVersion = async (check: boolean) => {
+    const { localConfig, rbRootDir } = await getLocalRbConfig();
+
+    if (!localConfig.version) {
+        console.log(chalk.red(`${Emoji.error} Error: RunBotics config file is invalid`));
+        process.exit(1);
+    }
+
+    const localVersion = localConfig.version;
+
+    if (!check) {
+        console.log(`${Emoji.skip} Skipping remote version sync`);
+        return [ localVersion, rbRootDir ];
```

Comment:
Sometimes u use spaces when destructuring array, sometime you dont, pick one

Reply:
eg line 25

Reply:
lack of eslint in the project :D

Reply:
now its there